lesson3/ToDoList2/app/src/main/java/erica/todolist2/SettingsFragment.java

```diff
@@ -21,12 +21,12 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         red.setOnClickListener(new View.OnClickListener() {
             @Override
             public void onClick(View view) {
-                View activityView = getActivity().findViewById(R.id.linear_layout);                 //defines the view whose background we want to set the color to
+                //defines the view whose background we want to set the color to
```

Comment:
Since this is a long comment, I would put it on it's own line above what it is referencing